mar_13_assignment/gonza.md

```diff
@@ -1,3 +1,25 @@
-## This is my homework
-# testing MD
-**Bold**
+## SUPREME COURT OF THE UNITED STATES
+### IN THE SUPREME COURT OF THE UNITED STATES
+
+
+Place: Washington, D.C.
+
+Date: March 12, 2050
+<br></br><br></br>
+Mr. Chief Justice, and may it please the Court:
+
+Since the implementation of the "Grab Them By The Uterus" Act, almost 19 years ago. According to it, every woman in Puerto Rico has received their pregnancy prosecution status after each period, by email. Whenever their Smart Tampon detects flow, User Data was put to work. If her life style and behaviors would prove at least 95% chance of the unborn to commit any crimes during their lifetime, the woman would be sentenced for instigation to criminalize their future son. In this way, as stated by the Act, every woman would know their current status on a monthly basis. If the status was *guilty* and decided to get pregnant anyway, it would be "their choice to do so".
+
+This resulted in an entire generation being born and raised inside jails, educated as criminals and only freed after their 18th birthday. This year marks the first anniversary of the first batch release.
+
+The People come once again to this court to communicate how displeased we are with the current situation. To begin with, the data center which analyses personality is not public but acts as a branch of the Corrections Corporation of America. So we can say that this private enterprise is on both ends of the business model. Moreover their prosecution status process is obscured to us. On their website, their algorithm is described as an "Efficient, effortless and smart way to simulate and predict the life of the unborn insurgent". As far as details go, we only have a part of a leaked presentation on which input for dataset is suggested: "Parents race, comparison between residency and crime map provided by State Police, education level of nearby schools and finer details on lifestyle thanks to agreement with Broadcom, worlds largest GPS provider".
+
+Your Honor, the People demand access to specifics about the way the algorithm works. It is of public interest to fully understand the way our actions are being judged, specially after the legal system was outsourced to software companies. It is logical to understand why these algorithms were hidden to public some 30 years ago, but nowadays people can read most program languages. We have no idea who and why traced this rules. We should be able to read the specific ways this works and let us judge wether decisions were biased or not.
+
+It is in our concern that the government of the United States would plan to execute a similar plan on US soil, using the puertorican experience as a way to practice before releasing to a wider audience. Dumb tampons and pads are almost non-existent and it is only possible to find the IoT tampons, leading some to speculate about the interests Market would have to force people into them.
```

Comment:
It is our concern that the government of the United States would plan to execute a similar plan on US soil, using the puertorican experience as a way to practice before releasing to a wider audience.